### PR TITLE
Remove clone'ing of metadata

### PR DESCRIPTION
**Pull request by zachthompson**
We don't really need to force read-only metadata.

### CODE REVIEW

File: lib/DDG/Meta/Data.pm [hunk 1]
```diff
@@ -152,12 +151,8 @@ sub apply_keywords {
         );
     }
     $s->add_symbol('&metadata', $id_required ? 
-        sub {
-            my $m = $dynamic_meta->($_[1]);
-            return clone($m);
-        }
-        :
-        sub { clone($ias->[0]) }
+        sub { return $dynamic_meta->($_[1]); } :
```
Comment (nilnilnil) [inline lib/DDG/Meta/Data.pm]: why a return here and semi here and none below? isn't it all the same?

Comment (nilnilnil) [inline lib/DDG/Meta/Data.pm]: just want to be consistent

**Comment by nilnilnil** [comment]
:+1: nits

**Comment by nilnilnil** [comment]
:+1:

**Comment by nilnilnil** [comment]
thx

**Comment by nilnilnil** [comment]
:godmode:

**Comment by russellholt** [comment]
:+1: thanks all

**Pull request merged**